code/modules/antagonists/traitor/contractor/datums/contractor_hub_ui.dm

```diff
@@ -0,0 +1,137 @@
+
+#define PAGE_CONTRACTS 1
+#define PAGE_HUB 2
+
+/datum/tgui_module/contractor_uplink
```

Comment:
Any reason this is a TGUI module instead of just being on the hub itself?

Also this should really be in `code/modules/tgui/modules`

Reply:
No specific reason besides keeping things separate. As for the second point, I personally prefer to keep as much as possible self-contained in its own module.